test/aries-js-worker/fixtures/docker-compose.yml

```diff
@@ -24,3 +24,19 @@ services:
       - ${E2E_ROUTER_WS_INBOUND_PORT}:${E2E_ROUTER_WS_INBOUND_PORT}
       - ${E2E_ROUTER_API_PORT}:${E2E_ROUTER_API_PORT}
     command: start
+
+  user.agent.example.com:
```

Comment:
nitpick: rename to `user.agent.rest.example.com` or something that similarly communicates how this is the agent-rest we're interfacing with

Reply:
All docker images are rest based. Only difference is while creating the Aries object in the js files (passing controller url(rest backend) v wasm pararm(wasm backend)).